src/Resources/ResourceManager/Implementation/DeploymentScripts/SaveAzDeploymentScriptLog.cs

```diff
@@ -118,11 +122,16 @@ public override void ExecuteCmdlet()
                     this.ConfirmAction(
                         this.Force || !AzureSession.Instance.DataStore.FileExists(outputPathWithFileName),
                         string.Format(
-                            Properties.Resources.DeploymentScriptLogFileExists, Name, OutputPath),
+                            Properties.Resources.DeploymentScriptLogFileExists, scriptName, OutputPath),
                         Properties.Resources.DeploymentScriptShouldProcessString,
                         OutputPath,
                         () =>
                         {
+                            //Standardize newline character to be written into file
+                            string[] delims = new string[] { "\r", "\n", "\r\n" };
```

Comment:
This is good but just curious did StringReader not work?

Reply:
I didn't really try StringReader. I wasn't sure if ReadLine() would consider all 3 possible line endings or only Environment.newline.

Reply:
It should consider all 3 and end of stream marker. See here: https://docs.microsoft.com/en-us/dotnet/api/system.io.stringreader.readline?view=netcore-3.1

Reply:
Oh! let me quickly try that out as well